Assets/Scripts/ListToButtonManager.cs

```diff
@@ -0,0 +1,76 @@
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ListToButtonManager : MonoBehaviour
+{
+
+    public delegate void ListEvent(int index);
+    public static ListEvent onChangeIndex;
+
+    [SerializeField] private Button forward;
+    [SerializeField] private Button back;
+    List<Cell> list = new();
+    private int index;
+
+
+    private int Index
+    {
+        get => index;
+        set
```

Comment:
I personally don't mind this style for a property, but just to let you know some people will argue that there is more logic here than simply getting or setting a variable, and should be a function as a result. 

Doesn't bother me either way just to let you know.

Also! did you know you can set access modifiers differently on the get and set calls? so your get could be public but your set can only be set within the class.

Reply:
Resolved